shared/utils/relabel-ids.py

```diff
@@ -206,13 +207,16 @@ def drop_oval_checks_extending_non_existing_checks(ovaltree, indexed_oval_defs):
 
             if referreddefinition is None:
                 # There is no oval satisfying the extend_definition referal
-                defstoremove.append((definition, extdefinitionref))
+                incomplete = True
+                sys.stderr.write("WARNING: OVAL check '%s' extends non-existing '%s' "
+                "OVAL definition, removing check from OVAL definitions.\n"
+                % (definition.get("id"), extdefinitionref))
```

Comment:
On the other hand, if having several extensions are not a common scenario, printing it for every error won't be a big deal.